### DISCUSSION TITLE
How to get rid of example pages/files in /docs

### DISCUSSION THREAD

**Discussion by sambraza** [wangchucheng/hugo-eureka#114] [Q&A]
Hi,

Apologies in advance for the dumb question. I'm using Eureka for my personal website, and doing it all in R Studio with R Blogdown.  I was trying to get rid of the example documents, like in /docs, but if I delete the files they get rebuilt. I tried  deleting the index.html/index.xml files, the example doc files, and deleting the whole docs folder, but every time I rebuild the site the folder gets regenerated. Not really sure where how to change this, or if it even can be changed. I did read through the documentation site, but maybe I was missing the answer.

**Comment by wangchucheng** [comment]
I do not use r studio. But I think if you delete the related files in content folder, they will not be rebuilt in the common Hugo way.

If there is anything that can help you, please feel free to tell me.


________________________________
发件人: sambraza ***@***.***>
发送时间: Friday, June 11, 2021 7:48:11 AM
收件人: wangchucheng/hugo-eureka ***@***.***>
抄送: Subscribed ***@***.***>
主题: [wangchucheng/hugo-eureka] How to get rid of example pages/files in /docs (#114)


Hi,

Apologies in advance for the dumb question. I'm using Eureka for my personal website, and doing it all in R Studio with R Blogdown. I was trying to get rid of the example documents, like in /docs, but if I delete the files they get rebuilt. I tried deleting the index.html/index.xml files, the example doc files, and deleting the whole docs folder, but every time I rebuild the site the folder gets regenerated. Not really sure where how to change this, or if it even can be changed. I did read through the documentation site, but maybe I was missing the answer.

―

**Reply by sambraza** [reply]
Thank you, that worked perfectly. Somehow I hadn't found that folder.
